### ISSUE TITLE
Pin columns when scroll horizontal is enabled

### ISSUE THREAD

**Issue by juanpablof** [ssuperczynski/ngx-easy-table#231]
It would be nice to pin/lock columns that shouldn't scroll horizontally (similar to Excel where you lock some columns to avoid losing reference when scrolling). For instance:

```ts
this.eventsColumns = [
  { key: 'action', title: 'Actions', pinned: true },
  { key: 'date', title: 'Date', pinned: false }
];
```

Alternatively, it would be great to use css directly in the column definition: 

```ts
this.eventsColumns = [
  { key: 'action', title: 'Actions', cssClass: 'column-1' },
  { key: 'date', title: 'Date', cssClass: 'column-2' }
];
```

**Comment by ssuperczynski** [comment]
Hi @juanpablof,
I remember I created PR with pinned column option some time ago, but caused some errors and didn't merge it. However it's time to bring this feature back, I'll try to add both options soon. Give me 1 week to figure it out.

**Comment by juanpablof** [comment]
That's great. Please, keep me posted. Pinned columns seem like a great feature for large data tables. The css column option isn't that critical, but it's a nice touch if you would like to highlight some columns to drive more attention. Thanks again!

**Comment by ssuperczynski** [comment]
I've finished implementing both features. I'll release it this week as v9.4.0

**Comment by juanpablof** [comment]
That’s great! Thank you Sebastian.

**Comment by juanpablof** [comment]
@ssuperczynski would you mind confirming when are you releasing v9.4.0? Thank you in advance!

**Comment by ssuperczynski** [comment]
Apologise for not releasing last week, still minor fixes need to be applied. I'll try to release it before end of this week.

**Comment by ssuperczynski** [comment]
Version 9.4.0-rc.1 has been released. Feature should be pretty stable now, but still couple things to fix before 9.4.0

https://ngx-easy-table.eu/#/pinned

**Comment by ssuperczynski** [comment]
Version 9.4.0 has been released https://github.com/ssuperczynski/ngx-easy-table/releases/tag/9.4.0.

Thanks got great feature idea 🚀
